src/test/regress/spec/am_vacuum_vs_insert.spec

```diff
@@ -1,5 +1,6 @@
 setup
 {
+    ALTER SEQUENCE cstore.cstore_storageid_seq RESTART WITH 205;
```

Comment:
Guess this is to compensate for the id being in the output. Would normalizing work? I forgot if they apply to the isolation tests